cypress/e2e/MainUserFlow.cy.ts

```diff
@@ -147,12 +147,13 @@ describe('Main user flow', () => {
     cy.get('[data-cy="Download-step"]').within(() => {
       cy.get('.MuiStepLabel-iconContainer').should('have.class', 'Mui-active');
     });
-    cy.get('[data-cy="complete-annotations-alert"]').should('be.visible');
+    cy.get('[data-cy="incomplete-annotations-alert"]').should('be.visible');
     cy.get('[data-cy="datadictionary-preview"]')
       .should('be.visible')
       .should('contain', 'Description:"some cool new description"');
     cy.get('[data-cy="datadictionary-toggle-preview-button"]').click();
     cy.get('[data-cy="datadictionary-preview"]').should('not.be.visible');
+    cy.get('[data-cy="force-download-switch"]').click();
```

Comment:
iiuc this test would not have failed previously, correct?

Reply:
Yes, that would've passed previously with a partially complete annotation.